app/views/components/InputPostalCodeAddress.scala.html

```diff
@@ -0,0 +1,89 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.Address
+@import models.reference.Country
+@import viewModels.components.InputSelectViewModel._
+@import viewModels.components.InputTextViewModel._
+@import views.html.components.{InputSelect, InputText}
+@import views.utils.ViewUtils._
+
+@this(
```

Comment:
Can we see a `a11y/components` spec for this component? Will be largely similar to https://github.com/hmrc/manage-transit-movements-departure-frontend/blob/main/a11y/components/InputAddressSpec.scala